### PR TITLE
AutoChooser and Runner

### LINKED ISSUE #9: Add AutoChooser + AutoRunner

### PR DESCRIPTION
**Pull request by ThatXliner**
- Closes #9 

Untested. I'm not even sure if this code would work

### CODE REVIEW

File: src/main/java/frc/robot/autos/pathPlannerAuto.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+package frc.robot.autos;
+
+import com.pathplanner.lib.auto.AutoBuilder;
+import com.pathplanner.lib.path.PathPlannerPath;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class pathPlannerAuto extends SequentialCommandGroup {
```
Comment (realjoshuau) [inline src/main/java/frc/robot/autos/pathPlannerAuto.java:17]: you should change this to proper casing

**Review by realjoshuau — COMMENTED** [review]
Seems good to me!

File: src/main/java/frc/robot/autos/pathPlannerAuto.java (hunk 1 above)
Comment (ThatXliner) [reply src/main/java/frc/robot/autos/pathPlannerAuto.java:17]: The `exampleAuto` was like that

**Comment by ChaoticChaosTheGreat** [comment]
Added spotless, accept my pr #19

**Comment by 3LucasZ** [comment]
Please post a video of this working in sim, before I can review.

**Review by 3LucasZ — CHANGES_REQUESTED** [review]
So far so good, but please post a video of this working in simulation before I can further review.

**Comment by ThatXliner** [comment]
Closed because the merge conflicts are crazy

**Pull request closed without merging**